app/components/UploadTrack/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { memo, useState } from 'react';
+import { Steps, Button } from 'antd';
+import styled from 'styled-components';
+import * as colors from '@app/themes/colors';
+import { TrackDetailsComponent } from '../TrackDetailsComponent/index';
+import UploadTrackBaseDetails from '../UploadTrackBaseDetails/index';
+import If from '../If/index';
```

Comment:
please use @components instead of relative paths

Reply:
also nothing wrong with `/index` but it reads a bit better without it. 

so `@components/TrackDetailsComponent` instead of `'../TrackDetailsComponent/index'`

Reply:
we need to do this for all of the imports

Reply:
If and TrackDetailsComponent too. 

Also as a practice we always import default exports instead of named imports for components. 


```
import { TrackDetailsComponent } from '../TrackDetailsComponent/index';
```
->
```
import TrackDetailsComponent from '@components/TrackDetailsComponent';
```